mopidy_spotify/translator.py

```diff
@@ -232,3 +232,27 @@ def _transform_year(date):
         logger.debug(
             'Excluded year from search query: '
             'Cannot parse date "%s"', date)
+
+
+def webapi_to_artist(sp_artist):
```

Comment:
`sp_` is a play on the `sp_` prefix of all C functions in libspotify. To make a clearer distinction between libspotify and Web API, maybe use `web_` prefix instead?